GPU/GPUTracking/Global/GPUChainTrackingTRD.cxx

```diff
@@ -37,6 +37,7 @@ int GPUChainTracking::RunTRDTracking()
   if (mIOPtrs.nTRDTracklets == 0) {
     return 0;
   }
+  Tracker.SetProcessPerTimeFrame(true);
```

Comment:
Do we want to enable this unconditionally? I mean, there is still the possibility that we process triggered events.
I think you should check that `GPUParam::par.continuousMaxTimeBin != 0` for time frames, = 0 means triggered event.

Reply:
Thanks I did not know that this option already existed. So I am using it now